src/modules/submissions.js

```diff
@@ -2364,7 +2364,8 @@
 
 						recentPage.edit( {
 							contents: newRecentText,
-							summary: 'Adding [[' + newPage + ']] to list of recent AfC creations'
+							summary: 'Adding [[' + newPage + ']] to list of recent AfC creations',
+							watchlist: 'nochange'
```

Comment:
Ditto here, this should never happen unless the user explicitly sets their preference to add all edited pages to watchlist.

Reply:
As I've mentioned elsewhere, even users that want to track all their edits on their watchlist are not going to want the `recentPage` re-added to their watchlist every time they accept a draft. It's a page that the user is never actually opting to update with the tool (it's an optional step in the manual reviewing instructions) that gets high enough traffic that it clogs up watchlists, and that is easy enough to watchlist manually if someone really wanted to.